examples/example-gateway/clients/quux/quux.go

```diff
@@ -21,16 +21,49 @@
 package quux
 
 import (
+	barClient "github.com/uber/zanzibar/examples/example-gateway/build/clients/bar"
 	module "github.com/uber/zanzibar/examples/example-gateway/build/clients/quux/module"
-	"github.com/uber/zanzibar/runtime"
+	zanzibar "github.com/uber/zanzibar/runtime"
+	"go.uber.org/zap"
+	"golang.org/x/net/context"
 )
 
 // Client is a custom client that does nothing yet
 type Client interface{}
 
-type quux struct{}
+type quux struct {
```

Comment:
I dont think this clients needs changing...

Reply:
Just using it as an example to demonstrate how to create a custom tchannel client, but I can remove it.